docs/features/pixel-art-converter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Pixel Art Converter
+hide:
+  # - navigation
+  # - toc
+---
+
+The Pixel Art Converter tool aims to make it easier to show pixelart on a LED matrix panel, by converting any image to 2D pixelart and send it to the WLED device.
+
+## Installation approaches
+
+There are two ways to install the pixel art converter:
+
+1. Local web browser. Will a web page run on the local machine connecting to the WLED device but will require to fetch an extra file. Supported from WLED release [v0.14.0-b1](https://github.com/Aircoookie/WLED/blob/main/CHANGELOG.md#wled-release-0140-b1) or later
+2. Include the pixel art converter in the binary and compile from source. Allows access the pixel art converter on any device that have a connection to the WLED device. Supported from WLED release [v0.14.0-b2](https://github.com/Aircoookie/WLED/blob/main/CHANGELOG.md#build-2301240) (PR [#3042](https://github.com/Aircoookie/WLED/pull/3042))
+
+### Approach 1: Local Browser
+The instruction for the pixel art generator is best described in the original repository under "Basic Operations": [https://github.com/werkstrom/WLED-PixelArtConverter/](https://github.com/werkstrom/WLED-PixelArtConverter/).
+
+### Approach 2: Include Pixart Converter in build files
+!!! warning "Compilation required"
+	Compiling WLED from source is required. Follow the instructions on [compiling WLED](advanced/compiling-wled) in order to do this.
+
+1. Set the flag `#define WLED_ENABLE_PIXART` in `./WLED/wled00/my_config.h`
```

Comment:
Please use PIO environment option `-D WLED_ENABLE_PIXART` as a preference to changing `my_config.h`

Reply:
I have changed the steps to use the PIO environment option instead. What is the reason for moving away my_config and using custom PIO environments?